src/meta/src/manager/catalog/mod.rs

```diff
@@ -1274,9 +1274,13 @@ impl CatalogManager {
                         continue;
                     }
 
-                    // add cdc source id
+                    // cdc source streaming job
```

Comment:
Any reason why we need an additional `all_cdc_source_ids ` instead of `all_source_ids`?

Reply:
`all_cdc_source_ids` is for recording stream job id for the cdc source job, then caller will call `drop_streaming_jobs` to drop actors for cdc source job.